### PR TITLE
Onevideo supporting both CCPA and GDPR present case

### PR DESCRIPTION
**Pull request by DeepthiNeeladri**
## Type of change

- [X ] Bugfix

## Description of change

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by bretg** [comment]
please add/modify unit test(s)

**Comment by DeepthiNeeladri** [comment]
@bretg added test cases, all are passing in my local but ci/circleci is failing  with error "Build was canceled"

**Comment by jsnellbaker** [comment]
@DeepthiNeeladri We're having some issues (for some reason) with browserstack keeping the browser sessions connected.  I tried to cancel the build since it was just restarting the sessions in a loop.  Trying to run it again.

**Comment by DeepthiNeeladri** [comment]
@bretg @jsnellbaker  review pls

File: test/spec/modules/oneVideoBidAdapter_spec.js [hunk 1]
```diff
@@ -221,9 +221,16 @@ describe('OneVideoBidAdapter', function () {
...
-          allowAuctionWithoutConsent: 'cancel'
+          gdpr: {
+            cmpApi: 'iab',
+            timeout: 3000,
+            allowAuctionWithoutConsent: 'cancel'
+          },
+          usp: {
+            cmpApi: 'iab',
+            timeout: 1000,
+            allowAuctionWithoutConsent: 'cancel'
```
Comment (bretg) [inline test/spec/modules/oneVideoBidAdapter_spec.js]: for the record, usp doesn't support the allowAuctionWithoutConsent parameter

**Comment by bretg** [comment]
Will go ahead and merge this, but note that you'll probably want to add a docs PR so you show as one of the adapters supporting CCPA. Set the `usp_supported` flag on https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/oneVideo.md

**Comment by bretg** [comment]
Also note -- so far it's only  merged into 3.1. If you want CCPA supported in the 2.x branch please open another PR against the 2.44.x-legacy branch

**Pull request merged**